run-tests.php

```diff
@@ -1575,15 +1575,16 @@ function run_test($php, $file, $env)
 
 	// Additional required extensions
 	if (array_key_exists('EXTENSIONS', $section_text)) {
-		$ext_dir=`$php -r 'echo ini_get("extension_dir");'`;
+		$ext_dir=`$php -r "echo ini_get('extension_dir');"`;
```

Comment:
Are you sure this doesn't hurt the non windows side? Some shells could extrapolate double quotes and consequently complain about parentheses. IMO it would be safer to isolate the call by platform.

Thanks.